lib/faktory/cli.rb

```diff
@@ -188,6 +188,12 @@ def boot_system
 
         require(options[:require]) || raise(ArgumentError, not_required_message)
       end
+
+      # We hit this during faktory-worker boot, it allows jobs that were queued
+      # with the ActiveJob adapter to be processed correctly.
+      if defined?(::ActiveJob)
+        require 'active_job/queue_adapters/faktory_adapter'
+      end
```

Comment:
This is redundant with the require in lib/faktory.rb.  cli.rb should require faktory.rb.

Reply:
@mperham I added this `require` because it seems that when you run `faktory-worker` that `ActiveJob` is not loaded by the time we hit the conditional `require` in lib/faktory.rb, which resulted in the adapter never being loaded and jobs being unable to process. Putting it here allows rails to load (along with AJ) so that everything "just works". Without this job classes would need to explicitly require the adapter, similar to what I found in my "stand alone" test. I'm sure you understand the entire loading process better than I do, so I'd welcome any ideas on how to get this working.